### PR TITLE
Portfolio Webiste (Desktop Vesrion Added)

### PR DESCRIPTION
**Pull request by umair-rao**
I have choosed template No.3 and added desktop version according to template.

- In this PR I have made Portfolio Webiste responsive for desktop version.
- Dimensions for Mobile vresion is (497x734).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -170,6 +173,7 @@ <h2 id="about-text">About me</h2>
             <button type="button" class="green-button">Get My Resume</button>
```
Comment (sajid-munawar) [inline index.html:180]: - [ ] Kindly make sure you apply button states that are for interactivity. The states are eg on hover, active and etc.
btnfigma
- #### Please apply for the rest of your buttons in your portfolio👍.

File: index.html [hunk 2]
```diff
@@ -12,14 +12,15 @@
 <div class="container">
     <header class="toolbar">
         <a href="index.html">
-            <img src="images/logo.svg" alt="Logo" />
+            <img id="logo-image" src="images/logo.svg" alt="Logo" />
         </a>
         <nav class="menu">
             <img src="images/menu.svg" alt="menu_icon">
         </nav>
         </header>
```
Comment (sajid-munawar) [inline index.html:27]: - [ ] Please add the navbar for the desktop version.
## Your Design👇
headerdeskst
 ## Figma design👇
headerfigma

File: index.html [hunk 3]
```diff
@@ -222,11 +226,13 @@ <h2 id="about-text">About me</h2>
...
+    </div>
     </section>
     <!-- About section end here  -->
 
     <!-- footer section starts from here -->
     <footer class="form">
+        <div id="form-detail">
         <div class="form-description">
             <h2 id="form-desc">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
         </div>
```
Comment (sajid-munawar) [inline index.html:245]: - [ ] Kindly make sure your desktop footer version is the same as the figma design. please follow the design and fix it as it is a core requirement and good for your designing skill.
## your design 👇
desktopst

## Figma design👇
desktopfigma

**Review by sajid-munawar — CHANGES_REQUESTED** [review]
# Changes Required 
 ### Hi @umair-rao 
 Nice job, you are almost there.
 
 
 
 ## (Highlights) Good Points: 
 * Good commit messages. :heavy_check_mark:
 * No linter errors. :heavy_check_mark:
 * Followed Github flow :heavy_check_mark:
 * Professional README file. :heavy_check_mark:


 ## (Changes Required) Aspects to improve: 
 * [ ]  Please check my inline comments.
 - [ ] Please kindly add a more descriptive PR title.
 See a demo below:

pull_request_desc
 

- [ ] Please make sure when we change from desktop version to mobile version padding and margins are set and the same as the figma design.
## your desing👇
mobilest

## Figma design👇
figmamobile


 
 ## [Optional] suggestions:
 * Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 **Cheers and Happy coding! :clap::clap::clap:**
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear
 Please, remember to tag @sajid-munawar in your question so I can receive the notification.

 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (umair-rao) [reply index.html:180]: Not included in my template (Hover or any other effect).

File: index.html (hunk 2 above)
Comment (umair-rao) [reply index.html:27]: Navbar is added according to template.

File: index.html (hunk 3 above)
Comment (umair-rao) [reply index.html:245]: Desktop footer is now according to template except the form part as it is written in description of task that we may have same type of form for mobile and desktop vesrion.

**Comment by umair-rao** [comment]
I have made all suggested changes please review. 
Thanks.

File: style.css [hunk 4]
```diff
@@ -303,3 +327,159 @@ input {
...
+    grid-template-columns: 50% 50%;
+    margin-top: 5%;
+    padding-left: 7%;
+  }
+
+  .form-description {
+    width: 100%;
+  }
+
+  #form-desc {
```
Comment (Ugboaja-Uchechi) [inline style.css:455]: - [ ] Like the previous reviewer mentioned kindly make sure your `footer` looks as close as possible to the Figma design especially the text. 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 5]
```diff
@@ -303,3 +327,159 @@ input {
...
+@media screen and (min-width: 768px) {
+  #headline-img-desktop {
+    display: block;
+    width: 100%;
+  }
+
+  .navbar {
+    display: block;
+    margin-left: 41%;
+  }
```
Comment (Ugboaja-Uchechi) [inline style.css:357]: - [ ] [OPTIONAL] Your site overflows. Kindly fix this 🙏

File: style.css [hunk 6]
```diff
@@ -303,3 +327,159 @@ input {
...
+
+  #textarea {
+    width: 400px;
+    height: 114px;
+    border-radius: 4px;
+    border-color: #60c095;
+  }
+
+  #get-in-btn {
```
Comment (Ugboaja-Uchechi) [inline style.css:478]: - [ ] Like the previous reviewer mentioned kindly give all your `buttons` interactions. 🙏 . Apologies for the wrong screenshot given by the previous reviewer

image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @umair-rao, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (umair-rao) [reply style.css:455]: Capture

Text corrected and updated according to template.

File: style.css (hunk 5 above)
Comment (umair-rao) [reply style.css:357]: Thank for suggestion it is also fixed now.

File: style.css (hunk 6 above)
Comment (umair-rao) [reply style.css:478]: Hover, and Pressed interactions are added for buttons.

**Comment by umair-rao** [comment]
All suggested changes have been made please check inline comments reply and project.

**Review by Olamarx — APPROVED** [review]
Hi @umair-rao, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good commit messages ✔️ 
- Linters passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**